.github/workflows/gpu.yaml

```diff
@@ -0,0 +1,34 @@
+name: GPU on manual trigger
```

Comment:
Just curious, how would you trigger this? Perhaps adding some small instructions for how to run the release tests would help here.

Reply:
I think there's supposed to be a button under the "Actions" tab here on GitHub after merging. Guess we'll find out!